packages/fxa-content-server/tests/functional/subscriptions.js

```diff
@@ -66,5 +68,39 @@ registerSuite('subscriptions', {
         .then(testElementExists(selectors['123DONE'].AUTHENTICATED))
         .then(visibleByQSA(selectors['123DONE'].SUBSCRIBED));
     },
+    'sign up, failed to subscribe due to expired CC': function () {
+      if (
+        process.env.CIRCLECI === 'true' &&
+        !process.env.SUBHUB_STRIPE_APIKEY
+      ) {
+        this.skip('missing Stripe API key in CircleCI run');
```

Comment:
Should we throw here instead to avoid any false positives in circle?

Reply:
This would mean that the tests fail for any PR not on the main repo (i.e. from a contributor or one of our personal forks)